docs/source/models/cause.rst

```diff
@@ -255,13 +254,13 @@ and discussed in more detail afterward.
      - Rate at which an individual accrues years lived with disability due to
        the state in the cause model.
    * - `Cause-specific Mortality`_
-     -
-     -
-     -
+     - Mortality rate from a specified cause for a population
+     - Mortality Impacts
+     - Measure of the frequency of occurrence of death in a defined population from a specified cause
```

Comment:
The CSMR is used as the reduction in the background mortality rate (producing the cause deleted mortality rate) among people living without a particular cause.